### PR TITLE
Enable NRT in another project.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs [hunk 1]
```diff
@@ -265,23 +267,16 @@ private static IfStatementSyntax GetOutermostIfStatementOfSequence(SyntaxNode if
...
-            {
-                if (!result.IsParentKind(SyntaxKind.ElseClause))
-                {
-                    break;
-                }
-
-                result = (IfStatementSyntax)result.Parent.Parent;
-            }
+            while (result.IsParentKind(SyntaxKind.ElseClause))
+                result = (IfStatementSyntax)result.GetRequiredParent().GetRequiredParent();
```
Comment (davidwengier) [inline src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs:279]: Might just be me, but I much rather the previous form as far as readability goes - and I'm not talking about the braces 😛

I'm not even sure my brain can work out if these are equivalent.

File: src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/CSharpUseRangeOperatorDiagnosticAnalyzer.cs [hunk 2]
```diff
@@ -92,35 +92,25 @@ protected override void InitializeWorker(AnalysisContext context)
...
             // want to make sure we're on something the fixer will know how to actually fix.
-            if (!(invocation.Syntax is InvocationExpressionSyntax invocationSyntax) ||
+            if (invocation.Syntax is not InvocationExpressionSyntax invocationSyntax ||
                 invocationSyntax.ArgumentList is null)
             {
                 return null;
             }
 
-            CodeStyleOption2<bool> option = null;
-            if (analyzerOptionsOpt != null)
```
Comment (davidwengier) [inline src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/CSharpUseRangeOperatorDiagnosticAnalyzer.cs:102]: Rename analyzerOptionsOpt to analyzerOptions?

**Review by davidwengier — APPROVED** [review]

File: src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs:279]: The problem with the prior form is that it was comparing something that could not be null to null. That informs the compiler the value is Nullable, which causes*more* warnings to deal with :-)

File: src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/CSharpUseRangeOperatorDiagnosticAnalyzer.cs (hunk 2 above)
Comment (CyrusNajmabadi) [reply src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/CSharpUseRangeOperatorDiagnosticAnalyzer.cs:102]: That's reasonable :-)

File: src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs [hunk 3]
```diff
@@ -290,21 +285,17 @@ private static IfStatementSyntax GetOutermostIfStatementOfSequence(SyntaxNode if
...
                 if (statement.Statement.IsKind(SyntaxKind.Block))
-                {
                     return true;
-                }
 
                 var elseStatement = statement.Else?.Statement;
                 if (elseStatement.IsKind(SyntaxKind.Block))
-                {
                     return true;
-                }
```
Comment (sharwell) [inline src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs:307]: 😢 I really wish this pull request only included the items it said it included

File: src/Analyzers/CSharp/Analyzers/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs [hunk 4]
```diff
@@ -84,13 +82,13 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
...
 
-            if (!innerStatement.IsKind(SyntaxKind.ExpressionStatement, out ExpressionStatementSyntax expressionStatement))
+            if (!innerStatement.IsKind(SyntaxKind.ExpressionStatement, out ExpressionStatementSyntax? expressionStatement))
             {
                 return;
             }
 
             // Check that it's of the form: "if (a != null) { a(); }
-            if (!(expressionStatement.Expression is InvocationExpressionSyntax invocationExpression))
+            if (expressionStatement.Expression is not InvocationExpressionSyntax invocationExpression)
```
Comment (sharwell) [inline src/Analyzers/CSharp/Analyzers/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs:91]: Lots of noise going on in this review. It's already large, so this makes it tremendously more time consuming to figure out which parts are actually needed and which parts are just change for the sake of change.

**Pull request merged**